src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
When I first looked at ClearD I thought it just clear bits without this epoch magic.
It was unique function, but nobody used it in such way.
For example:
flowgraph.cpp:        
```
// Initialize the per-block bbReach sets. (Note that we can't just call BlockSetOps::ClearD()
// when re-running this computation, because if the epoch changes, the size and representation of the
// sets might change).
```
So I prefer not to create the second function, that will confuse us in the future, but to replace the first.

However: liveness, lsra, regalloc, assertionprop, copyprop and emit  do not change the epoch.
flowgraph does, but it call MakeEmpty before ClearD.